tests/scalability/run_suite.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo Cloning cht-core to /cht-core
+git clone --single-branch --branch scalability-automation https://github.com/medic/cht-core.git;
```

Comment:
NB: branch name needs to change (obviously). I presume we can pass this as a param from GH Actions?

Reply:
Yeah, this should be the beta tag I assume. I was using my branch specifically because I had some code and csvs. Would be passed similarly as how I'm applying the branch in another area. Will update.